package.json

```diff
@@ -196,12 +196,12 @@
     "browserslist-useragent-regexp": "4.1.3",
     "chai": "5.1.1",
     "del": "7.1.0",
-    "eslint": "8.57.1",
+    "eslint": "eslint/eslint",
```

Comment:
🤔 weird version number

Reply:
Yes, awaiting a new release somewhere next week. Now it points to their github repo and takes in latest dev.

Reply:
The new version has been released (9.12.0) so this is now replaced.